### PR TITLE
Add third_party directory and guideline.

### PR DESCRIPTION
**Pull request by odashi**
This pull request adds `explainaboard/third_party` directory, which is to be used for manual vendoring of third-party code. I also wrote a README that describes how to maintain this directory.

### CODE REVIEW

File: explainaboard/third_party/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Third-party libraries
+
+This directory contains libraries manually imported from other place.
+Code in this directory is maintained almost under the *as-is* basis. We don't guarantee
+that code is maintained under the same maintenance policy we applied to other parts in
+this repository.
+We also don't guarantee that the code is up-to-date.
+
+Before considering to upload any code to this directory, you have to make sure that the
```
Comment (pfliu-nlp) [inline explainaboard/third_party/README.md:9]: > consider uploading

File: explainaboard/third_party/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+change satisfies **all** of following conditions:
```
Comment (pfliu-nlp) [inline explainaboard/third_party/README.md:10]: `of the following`

File: explainaboard/third_party/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- If the code is trivial enough, consider implementing a similar logic to our main
+  component.
+- Code is distributed under a valid open-source license. The license have to be
+  compatible with the MIT license, which we applied to this repository.
+- Code is not able to be obtained from the PyPI.
```
Comment (pfliu-nlp) [inline explainaboard/third_party/README.md:16]: > The license have to be compatible with the MIT license
* `have` -> `has`
* could we elaborate more on `compatible`? For example, how do contributors know whether their code is compatible with MIT?

File: explainaboard/third_party/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+If you decided to upload a third-party to this directory, please follow the steps below:
+
+0. Don't use neither `git submodule` nor `git subtree` without special approvals.
+1. Make an individual directory to store the new third-party.
+2. Put code and license file onto the directory.
+3. If the code contains `README.md`, rename it to `README.orig.md`.
+4. Add `README.md`. It have to contain following information:
```
Comment (pfliu-nlp) [inline explainaboard/third_party/README.md:24]: >It have to contain following information:

* `have` -> has
* `following` -> the following

**Review by pfliu-nlp — APPROVED** [review]
Overall it looks good! Thanks for setting this up. We can refine it gradually when we have new things to say.

File: explainaboard/third_party/README.md (hunk 3 above)
Comment (odashi) [reply explainaboard/third_party/README.md:19]: Thanks, I added some ways to judge the compatibility.

**Review by pfliu-nlp — APPROVED** [review]
looks good!

**Pull request merged**